tsdb/wlog/wlog.go

```diff
@@ -201,12 +201,12 @@ type walMetrics struct {
 	writesFailed    prometheus.Counter
 }
 
-func newWALMetrics(r prometheus.Registerer) *walMetrics {
-	m := &walMetrics{}
+func newWLMetrics(r prometheus.Registerer) *wlMetrics {
+	m := &wlMetrics{}
 
 	m.fsyncDuration = prometheus.NewSummary(prometheus.SummaryOpts{
 		Name:       "fsync_duration_seconds",
-		Help:       "Duration of WAL fsync.",
+		Help:       "Duration of WL fsync.",
```

Comment:
WL might be confusing to the users. How about we say "write log" in all help text here?